python/tests/test_tables.py

```diff
@@ -2109,6 +2109,107 @@ def test_sequence_length_longer_than_edges(self):
         self.assertEqual(len(tree.parent_dict), 0)
 
 
+class TestTableCollectionMetadata(unittest.TestCase):
+
+    metadata_schema = metadata.MetadataSchema(
+        {
+            "codec": "json",
+            "title": "Example Metadata",
+            "type": "object",
+            "properties": {
+                "one": {"type": "string"},
+                "two": {"type": "number"},
+                "three": {"type": "array"},
+                "four": {"type": "boolean"},
+            },
+            "required": ["one", "two", "three", "four"],
+            "additionalProperties": False,
+        },
+    )
+
+    def metadata_example_data(self):
+        try:
```

Comment:
I don't like this as it makes the tests stateful and the test data will depend on what other tests are run on this class. Also adding instance variables to a TestCase is a bit weird. Either put in some constants, or make ``val`` a parameter.

Reply:
I made it a param.